### DISCUSSION TITLE
Prevent tiered Storage volumes delete

### DISCUSSION THREAD

**Discussion by Hower-Pan** [seaweedfs/seaweedfs#2566] [Q&A]
version 30GB 2.64 95e2b83ca5e8c23a775b4cda6519fc8e5f1161af linux amd64
ARCH:x86

I have 4 nodes on which weed were running, three node replication (001) 
1 node is remote site node that is tiered Storage
schedule use weed backup and copy to remote site node 
how to prevent remote site node volumes not delete by replication handle.   thank you :)

**Comment by chrislusf** [comment]
Please show the commands to start up these servers and backup scripts.

**Reply by Hower-Pan** [reply]
weed backup -server=master -dir=/tmp -volumeId=$volumeId -collection=$collectionid
cp /tmp/$filetag.idx remotesite:/vol
cp /tmp/$filetag.vif remotesite:/vol
cp /tmp/$filetag.dat remotesite:/vol

restart remote site weed volume service

thank you :)

**Reply by chrislusf** [reply]
"how to prevent remote site node volumes not delete by replication handle" I do not quite understand this sentence.

The source volume having deletions will need to run compaction.
So `weed backup` needs to delete the old backup copy and copy again.

**Reply by Hower-Pan** [reply]
when restart remote site weed volume service finished <<  i  think is not good , should be usr weed mount command
i saw backup volume is mount on tiered Storage node
volume server is deletes backup mount volume  during 3 min
so, i guess is replication issue
but if compaction make deletes backup mount volume 
why delete it not del source volume ??

       thank you :)

**Reply by chrislusf** [reply]
sorry, tried my best but could not fully understand your question.

the source volume is also compacted.
